src/App.jsx

```diff
@@ -2,34 +2,44 @@ import { useState } from 'react'
 import reactLogo from './assets/react.svg'
```

Comment:
Go ahead a remove code that isn't being used.

Reply:
will do